From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Doe <john.doe@somewhere.on.planet>
Date: Mon, 5 Feb 2024 09:10:47 +0800
Subject: Patching kernel rockchip64 files drivers/gpu/drm/panel/panel-edp.c

Signed-off-by: John Doe <john.doe@somewhere.on.planet>
---
 drivers/gpu/drm/panel/panel-edp.c | 29 ++++++++++
 1 file changed, 29 insertions(+)

diff --git a/drivers/gpu/drm/panel/panel-edp.c b/drivers/gpu/drm/panel/panel-edp.c
index 7dc6fb7308ce3..b9cd144d799e4 100644
--- a/drivers/gpu/drm/panel/panel-edp.c
+++ b/drivers/gpu/drm/panel/panel-edp.c
@@ -1689,10 +1689,36 @@ static const struct panel_desc starry_kr122ea0sra = {
 		.enable = 50,
 		.unprepare = 10 + 500,
 	},
 };
 
+static const struct drm_display_mode smilx_xjqh133ea3s_mode = {
+	.clock = 150000,
+	.hdisplay = 1920,
+	.hsync_start = 1920 + 3,
+	.hsync_end = 1920 + 3 + 23,
+	.htotal = 1920 + 3 + 23 + 5,
+	.vdisplay = 1080,
+	.vsync_start = 1080 + 48,
+	.vsync_end = 1080 + 48 + 80,
+	.vtotal = 1080 + 48 + 80 + 32,
+	.flags = DRM_MODE_FLAG_PHSYNC | DRM_MODE_FLAG_PVSYNC,
+};
+
+static const struct panel_desc smilx_xjqh133ea3s = {
+	.modes = &smilx_xjqh133ea3s_mode,
+	.num_modes = 1,
+	.size = {
+		.width = 309,
+		.height = 174,
+	},
+	.delay = {
+		.prepare_to_enable = 20,
+		.enable = 20,
+	},
+};
+
 static const struct of_device_id platform_of_match[] = {
 	{
 		/* Must be first */
 		.compatible = "edp-panel",
 	}, {
@@ -1777,10 +1803,13 @@ static const struct of_device_id platform_of_match[] = {
 		.compatible = "sharp,lq140m1jw46",
 		.data = &sharp_lq140m1jw46,
 	}, {
 		.compatible = "starry,kr122ea0sra",
 		.data = &starry_kr122ea0sra,
+	}, {
+		.compatible = "smilx,xjqh133ea3s",
+		.data = &smilx_xjqh133ea3s,
 	}, {
 		/* sentinel */
 	}
 };
 MODULE_DEVICE_TABLE(of, platform_of_match);
-- 
Created with Armbian build tools https://github.com/armbian/build

